testing/smoke-ide-test/build.gradle.kts

```
@@ -17,6 +15,17 @@ val smokeIdeTestSourceSet = sourceSets.create("smokeIdeTest") {
 addDependenciesAndConfigurations("smokeIde")
 val smokeIdeTestImplementation: Configuration by configurations
 val smokeIdeTestDistributionRuntimeOnly: Configuration by configurations
+val ideStarter by configurations.creating {
+    isVisible = false
+    isCanBeConsumed = false
+}
+
+val ideStarterHome = layout.buildDirectory.dir("ideStarter")
+
+val unzipIdeStarter = tasks.register<ProcessResources>("unzipIdeStarter") {
+    from(zipTree(provider { ideStarter.files.first() }))
```

💅 Keep the file collection chain intact. Be precise about the number of elements.
```suggestion
    from(zipTree(ideStarter.elements.map { it.single() }))
```